Objects/memoryobject.c

```diff
@@ -381,8 +381,9 @@ copy_rec(const Py_ssize_t *shape, Py_ssize_t ndim, Py_ssize_t itemsize,
 
 /* Faster copying of one-dimensional arrays. */
 static int
-copy_single(const Py_buffer *dest, const Py_buffer *src)
+copy_single(PyMemoryViewObject *self, const Py_buffer *dest, const Py_buffer *src)
 {
+    CHECK_RELEASED_INT(self); /* See gh-92888 for why we need this here */
```

Comment:
Can you move the check after the equiv_structure() test?

Reply:
Would not access to Py_buffer of the released object cause reading after free?